### PR TITLE
Added access to mentor tab for mentors.

### LINKED ISSUE #752: Mentors tab in WHAT UI Panel
Добавить функционал для ментора просмотра других менторов

### PR DESCRIPTION
**Pull request by BobTheSuperman**

### CODE REVIEW

**Review by DmitryTkachenko — APPROVED** [review]

File: CharlieBackend.Panel/Controllers/MentorsController.cs [hunk 1]
```diff
@@ -1,26 +1,39 @@
...
+        public MentorsController(IMentorService mentorService, ICurrentUserService currentUserService)
         {
             _mentorService = mentorService;
+            _currentUserService = currentUserService;
         }
 
         public async Task<IActionResult> AllMentors()
         {
-            var mentor = await _mentorService.GetAllMentorsAsync();
+            IEnumerable<MentorViewModel> mentor = null;
```
Comment (ArsenMaistryshyn) [inline CharlieBackend.Panel/Controllers/MentorsController.cs]: please return empty list

**Review by Ssashaa — APPROVED** [review]

File: CharlieBackend.Panel/Services/MentorService.cs [hunk 2]
```diff
@@ -77,6 +77,11 @@ public async Task<IList<MentorViewModel>> GetAllActiveMentorsAsync()
             var activeMentors = await
                 _apiUtil.GetAsync<IList<MentorViewModel>>(activeMentorEndpoint);
 
+            foreach(var mentor in activeMentors)
+            {
+                mentor.IsActive = true;
```
Comment (ArsenMaistryshyn) [inline CharlieBackend.Panel/Services/MentorService.cs]: please check IsActive property. Investigate ActiveMentorEndpoint.

**Review by ArsenMaistryshyn — CHANGES_REQUESTED** [review]

File: CharlieBackend.Panel/Services/MentorService.cs (hunk 2 above)
Comment (BobTheSuperman) [reply CharlieBackend.Panel/Services/MentorService.cs]: thx

File: CharlieBackend.Panel/Controllers/MentorsController.cs (hunk 1 above)
Comment (BobTheSuperman) [reply CharlieBackend.Panel/Controllers/MentorsController.cs]: thx

**Review by ArsenMaistryshyn — APPROVED** [review]

**Review by marinapelin — APPROVED** [review]

**Review by apyura — APPROVED** [review]

**Pull request merged**